docs/src/appendix/cheatcodes/load.md

```diff
@@ -0,0 +1,85 @@
+# `load`
+
+> `fn load(target: ContractAddress, storage_address: felt252, size: felt252) -> Array<felt252> `
+
+Loads `size` felts from `target` contract's storage into an `Array`, starting at `storage_address`.
+
+- `target` - Address of the contract, which storage you want to modify
+- `storage_address` - Offset of the data in the contract's storage 
+- `size` - How many felts will be loaded into the result `Array` 
+
+
+## Example usage
+
+```rust
+mod Contract {
+    #[storage]
+    struct Storage {
+        plain_felt: felt252,
+        mapping: LegacyMap<felt252, felt252>,
+    }
+}
+
+// ...
+use snforge_std::{ load, map_entry_address };
+
+#[test]
+fn load_plain_felt() {
+    // ...
+    
+    let loaded = load(contract_address, selector!("plain_felt"), 1);
+    assert(loaded.len() == 1, 'Wrong loaded vector');
+    assert(*loaded.at(0) == 0, 'Wrong loaded value');
+}
+
+#[test]
+fn load_map_entry() {
+    // ...
+    load(
+        contract_address, 
+        map_entry_address(
+            selector!("mapping"), // Providing variable name
+            array![123].span(),   // Providing mapping key 
+        ),
+        1,
+    );
+    
+    assert(loaded.len() == 1, 'Expected 1 felt loaded');
+    assert(*loaded.at(0) == 0, 'Expected 0 value loaded');
+}
+
+#[derive(starknet::Store)]
+struct ElaborateStructure {
+    a: felt252,
+    b: felt252,
+    // ...
+}
+
+#[test]
+fn load_elaborate_struct() {
+    // ...
+    load(
+        contract_address, 
+        map_entry_address(
+            selector!("mapping"), // Providing variable name
+            array![123].span(),   // Providing mapping key 
+        ),
+        StoreElaborateStructure::size().into(), // Auto-generated trait for starknet::Store deriving structs
+    );
+    
+    assert(loaded.len() == 1, 'Expected 1 felt loaded');
+    assert(*loaded.at(0) == 0, 'Expected 0 value loaded');
+}
+```
+
+> 📝 **Note** 
+> 
+> The cheatcode will return 0s for uninitialized memory, which is a default storage value for Starknet contracts.
```

Comment:
```suggestion
> The cheatcode will return zeros for "uninitialized" memory, which is a default storage value for Starknet contracts.
```
https://docs.starknet.io/documentation/architecture_and_concepts/Smart_Contracts/contract-storage/#storage_layout
If it was actually unititialized, it wouldn't be set to zeros ☝️🤓